package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "manh-react-survey",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "axios": "^1.4.0",
+    "i18next": "^22.5.1",
```

Comment:
Do we have a plan to use `i18n` in this project, if no, please remove this and the below two packages as well